dist/css/other_assets.css

```diff
@@ -5,3 +5,7 @@
     padding: 0;
     background: url('/assets/original/equipment-dude.jpg') no-repeat;
 }
+
```

Comment:
Wow... I did not know this. O_O seems like a bit of an oversight. I haven't really considered styling too much yet, might change to mdl or totally elm-css to get the old school look/feel at the end. atm I'm not too concerned with the color so happy for you to leave the css in there for now.

However, I would prefer to consolidate the API calls. ... Hey, @djuretic are you on elm-slack? We should have a chat if you're keen on doing more. I never really expected anyone to be adding stuff, but if you want to add more things, I think there should be some coordination.